ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp

```diff
@@ -564,6 +559,19 @@ class TSpillingSupportState : public TComputationValue<TSpillingSupportState> {
         return false;
     }
 
+    void LogMemoryUsage() const {
+        const auto used = TlsAllocState->GetUsed();
+        const auto limit = TlsAllocState->GetLimit();
+        TStringBuilder logmsg;
+        logmsg << "Memory usage: ";
+        if (limit) {
+            logmsg << (used*100/limit) << "%=";
+        }
+        logmsg << (used/1_MB) << "MB/" << (limit/1_MB) << "MB";
+
+        YQL_LOG(INFO) << logmsg;
```

Comment:
Maybe we want to print it on Debug level?

Reply:
Discussed that this log would be written once per task, so we decided to leave the INFO level.